README.md

```diff
@@ -42,14 +42,13 @@ To get started on **Visual Studio 2022**:
   - .NET desktop development
   - .NET Core cross-platform development
 2. Ensure [long path support](https://learn.microsoft.com/windows/win32/fileio/maximum-file-path-limitation?tabs=registry#enable-long-paths-in-windows-10-version-1607-and-later) is enabled at the Windows level.
-3. Open a `Developer Command Prompt for VS 2022` prompt.
-4. Clone the source code: `git clone https://github.com/dotnet/msbuild`
+3. [Install The Latest .NET SDK](https://dotnet.microsoft.com/en-us/download/dotnet)
+  - This will ensure you have the correct ASP.NET runtime packs.
```

Comment:
This should not be required (the one you get from VS should be sufficient). Did it appear to be?

Reply:
Yes, @edvilme encountered some issues even after installing the workloads, and the 4.5. (2?) workloads, he was still facing some runtime pack errors which this fixed

Reply:
We might have caused that by fixing one of the earlier errors 🥴 But anyways I'll remove the suggestion, thanks!